aloha/real_env.py

```diff
@@ -134,35 +147,58 @@ def setup_base(self, node: InterbotixRobotNode, torque_enable: bool = False):
     def setup_robots(self):
         setup_follower_bot(self.follower_bot_left)
         setup_follower_bot(self.follower_bot_right)
-
+    
     def get_qpos(self):
-        left_qpos_raw = self.recorder_left.qpos
-        right_qpos_raw = self.recorder_right.qpos
-        left_arm_qpos = left_qpos_raw[:6]
-        right_arm_qpos = right_qpos_raw[:6]
-        left_gripper_qpos = [FOLLOWER_GRIPPER_POSITION_NORMALIZE_FN(left_qpos_raw[7])]
-        right_gripper_qpos = [FOLLOWER_GRIPPER_POSITION_NORMALIZE_FN(right_qpos_raw[7])]
-        return np.concatenate(
-            [left_arm_qpos, left_gripper_qpos, right_arm_qpos, right_gripper_qpos]
-        )
+        # Initialize a list to hold the arm and gripper positions
+        qpos_list = []
+
+        # Iterate through all follower robots in the self.robots dictionary
+        for name, bot in self.robots.items():
+            if "follower" in name:
+                # Get the arm joint positions (first 6 joints)
+                arm_qpos = bot.core.joint_states.position[:6]
```

Comment:
I have added methods to the arm and gripper modules that can be used to get joint positions: https://github.com/Interbotix/interbotix_ros_toolboxes/tree/xs_robot_get_js

Go ahead and use those here instead of hardcoding the number of joints.